interfaces/builtin/all.go

```diff
@@ -34,6 +34,7 @@ var allInterfaces = []interfaces.Interface{
 	&NetworkManagerInterface{},
 	&PppInterface{},
 	&SerialPortInterface{},
+	&Tpm12Interface{},
```

Comment:
Why do we name it Tpm12? Shouldn't this be just TPM so its not specific to any software or its version?